test/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,7 @@
 const { humanCatDogYears } = require("../src");
-
 // Look Ma, no handlebars!!!
+describe("humanCatDogYears", () => {
```

Comment:
Maybe this test suite could be a little more robust by testing for all possible conditional branches.